modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+// * upgrade/
```

Comment:
Either needs removed (doesn't look like it's included in an upgrade assembly) or finished.